src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkUniIslService.java

```diff
@@ -139,6 +140,8 @@ private void handleDiscoveryFail(Endpoint endpoint, IslDownReason downReason) {
         IslReference reference = lookupEndpointData(endpoint);
         if (isIslReferenceUsable(reference)) {
             carrier.notifyIslDown(endpoint, reference, downReason);
+        } else if (reference.isIncomplete()) {
+            carrier.slowPollUpdateRequest(endpoint, true);
```

Comment:
... is just one fail response enough to switch endpoint into "slow" poll mode? I thought there should be series of time window without response for this.

Reply:
This is enough for our case (for ports not belonging to ISL). This has been tested and works as expected.